lcc_comchain_base/views/account_invoice_view.xml

```diff
@@ -14,8 +14,14 @@
                     numeric currency to credit for this customer. Please use mobile app to validate
                     credit request. </div>
             </xpath>
+            <xpath expr="//page[@name='other_info']/group" position="inside">
+                <group name="credit_data" string="Monnaie numérique">
+                    <field name="comchain_amount_to_credit" string="A créditer" readonly="True" />
+                    <field name="comchain_amount_credited" string="Crédité" readonly="True" />
```

Comment:
is that okay to use french here ? And BTW, thank you so much for these debugging info !.